benchmarks/test_peer_selection.py

```diff
@@ -39,6 +39,7 @@ class FakeConnection(object):
     def __init__(self, hostport):
         self.hostport = hostport
         self.closed = False
+        self.direction = object()
```

Comment:
Yeah, then do set it to INCOMING/OUTGOING. Makes it obvious what you're trying to do there.